src/js/redux/reducers/covid19/covid19Reducer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * covid19Reducer.js
+ * Created by Jonathan Hill 06/11/20
+ */
+
+const initialState = {
+    codes: []
+};
+
+const covid19Reducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'SET_CODES': {
```

Comment:
![image](https://user-images.githubusercontent.com/12897813/84436243-1a098e00-ac01-11ea-8b9a-036500bcb42f.png)


M O R E D E T A I L

Reply:
lol, i dont think there are any other codes and we are in the `covid19Reducer` What more do you want? 

Would `SET_DEF_CODES` work? of `SET_DEFC_CODES`? not sure if the `c` is for codes

Reply:
Yep, C is for code